pom.xml

```diff
@@ -42,8 +42,8 @@
         <utilities.version>1.17.0-M2</utilities.version>
         <test-utils.version>1.22.0-M1</test-utils.version>
         <framework.version>6.0.0-M20</framework.version>
-        <common-bom.version>1.28.0</common-bom.version>
-        <framework-api.version>4.0.0-M13</framework-api.version>
+        <common-bom.version>1.29.0</common-bom.version>
```

Comment:
Can the common-bom version be updated to 2.0.2

Reply:
common-bom updated to 2.0.2 with latest push.